src/Tasks/AssemblyDependency/ReferenceTable.cs

```diff
@@ -1343,9 +1340,7 @@ out userRequestedSpecificFile
         /// </summary>
         internal void RemoveReferencesMarkedForExclusion(bool removeOnlyNoWarning, string subsetName)
         {
-#if (!STANDALONEBUILD)
-            using (new CodeMarkerStartEnd(CodeMarkerEvent.perfMSBuildRARRemoveFromExclusionListBegin, CodeMarkerEvent.perfMSBuildRARRemoveFromExclusionListEnd))
-#endif
+            MSBuildEventSource.Log.RemoveReferencesMarkedForExclusionStart();
```

Comment:
Let's set up (and document) a convention to add a prefix for task events. I'd say to call this `RarRemoveReferencesMarkedForExclusion`.

Reply:
Isn't that "Remove Assembly References Remove References Marked for Exclusion" when written out? I could do `RarMarkedForExclusion` or `RemoveAssemblyReferencesMarkedForExclusion` if you'd like. (I prefer the latter.)

Reply:
I'd punctuate the expansion: "Remove Assembly References**:** Remove References Marked for Exclusion". I prefer that to your names.

Reply:
I see. Makes me want to put Rar:RemoveReferencesMarkedForExclusion or RemoveReferencesMarkedForExclusion(Rar), but then it wouldn't be a valid name.

I can see the value of having Rar on the front, though, since it would make it easy to find what has to do with RAR without looking through the whole list. I'll feel stupid if anyone asks me about it, but I think it's the best solution.

Reply:
Why would you feel stupid? If we're using a flat namespace here (methods in a single class) prefixing as poor-dev's-namespace makes perfect sense to me.

Reply:
Oh, I agree it makes sense. Not knowing the logic behind it, I would assume I had prepended Rar without actually knowing what it was, just knowing what it's associated with. Not true here, but I would criticize myself for using an acronym without knowing what it stood for and redundancy. With multiple rar methods, though, I do think it's important here for sorting/searching without ambiguity.